app/lib/synthetic_branding_group_populator.rb

```diff
@@ -0,0 +1,107 @@
+##
+# SyntheticBrandingGroupPopulator: class to populate branding groups using the output of SyntheticStudyPopulator
+##
+
+class SyntheticBrandingGroupPopulator
+  BASE_BRANDING_GROUPS_CONFIG_PATH = Rails.root.join('db', 'seed', 'synthetic_branding_groups')
+
+  # populate all branding groups using synthetic studies
+  #
+  # @param user (User)                 => User account to own the branding groups
+  # @param overwrite_groups (Boolean)  => T/F for destroying/recreating any existing groups
+  # @param overwrite_studies (Boolean) => T/F for destroying/recreating any existing studies
+  def self.populate_all(user: User.first, overwrite_groups: true, overwrite_studies: false)
```

Comment:
It would be nice to have an option to not populate any new studies on creation--basically 'just give me the branding groups, and I don't care if they are empty (or maybe I want them to be empty to test that particular case). then If that option is specified, we shouldn't error if a branding group has no study ids found.

Reply:
568c460, 7290786